### PR TITLE
Update LDAP auth docs

### PR DESCRIPTION
**Pull request by marcogschmidt**
Updated the LDAP extauth docs to use the latest configuration format.

### CODE REVIEW

File: gloo/docs/gloo_routing/virtual_services/security/ldap/_index.md [hunk 1]
```diff
@@ -74,17 +72,12 @@ spec:
...
-{{< tabs >}}
-{{< tab name="yaml" codelang="yaml">}}
+{{< highlight yaml "hl_lines=16-20" >}}
 {{< readfile file="gloo_routing/virtual_services/security/ldap/vs-echo-no-auth.yaml">}}
-{{< /tab >}}
-{{< tab name="glooctl" codelang="shell">}}
-glooctl create vs --name http-echo --namespace gloo-system
-glooctl add route --path-prefix /echo --dest-name default-http-echo-5678
-{{< /tab >}}
-{{< /tabs >}}
```
Comment (kdorosh) [inline gloo/docs/gloo_routing/virtual_services/security/ldap/_index.md]: why drop the `glooctl` options?

Comment (kdorosh) [reply gloo/docs/gloo_routing/virtual_services/security/ldap/_index.md]: If we aren't going to keep copy-pasteable `glooctl` commands, then we should make the virtual service copy-pasteable (i.e., k apply -f - << EOF ...)

Comment (marcogschmidt) [reply gloo/docs/gloo_routing/virtual_services/security/ldap/_index.md]: We currently don't support the new auth config format in the CLI :/
I'll update to something copy-pasteable.

File: gloo/docs/gloo_routing/virtual_services/security/ldap/_index.md [hunk 2]
```diff
@@ -250,12 +243,16 @@ dn: cn=managers,ou=groups,dc=solo,dc=io
...
+{{< readfile file="/static/content/extauth_version_info_note" >}}
+{{% /notice %}}
+
 Now that we have all the necessary components in place, let use the LDAP server to secure the Virtual Service we created 
 earlier .
 
 #### LDAP auth flow
 Before updating our Virtual Service, it is important to understand how Gloo interacts with the LDAP server. Let's first 
-look at the [LDAP auth configuration](https://gloo.solo.io/api/github.com/solo-io/gloo/projects/gloo/api/v1/enterprise/plugins/extauth/extauth.proto.sk/#ldap):
+look at the [LDAP auth configuration]({{< ref "/api/github.com/solo-io/gloo/projects/gloo/api/v1/enterprise/plugins/extauth/v1/extauth.proto.sk#ldap" >}}):
```
Comment (kdorosh) [inline gloo/docs/gloo_routing/virtual_services/security/ldap/_index.md]: 👏

File: gloo/docs/gloo_routing/virtual_services/security/ldap/vs-echo-no-auth.yaml [hunk 3]
```diff
@@ -13,6 +13,8 @@ spec:
...
       routeAction:
         single:
-          upstream:
-            name: default-http-echo-5678
-            namespace: gloo-system
+          kube:
+            ref:
+              name: http-echo
+              namespace: default
+            port: 5678
```
Comment (kdorosh) [inline gloo/docs/gloo_routing/virtual_services/security/ldap/vs-echo-no-auth.yaml]: OOC, why the change from the discovered upstream to a direct kube service ref?

Comment (marcogschmidt) [reply gloo/docs/gloo_routing/virtual_services/security/ldap/vs-echo-no-auth.yaml]: I thought since we just created a kube service and it reads:
> Now we can create a Virtual Service that routes any requests with the `/echo` prefix to the `http-echo` service

that it would be more appropriate to use the kube service destination. In the previous tutorials we use static upstreams, so it is more intuitive to use the upstream destination. This way we also showcase both features. :)
